ambry-network/src/main/java/com.github.ambry.network/Selector.java

```diff
@@ -72,6 +72,8 @@
   private final List<NetworkReceive> completedReceives;
   private final List<String> disconnected;
   private final List<String> connected;
+  private final List<Transmission> pendingSslHandshakes;
```

Comment:
Since only `SSLTransmission` will be added to this list, would it make more sense to specify? Of course, it would be needed to cast a `Transmission` into an `SSLTransmission` when you are actually handling it. Another argument to support this change is, though `Transmission.ready()` is called, but it is actually concerning the ssl aspect of the transmission for this particular case, so making it as `SSLTransmission.ready()` would make the code more readable.